Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1324,9 +1325,17 @@ void CFrame::ParseHotkeys()
 
 	if (!Core::IsRunningAndStarted())
 	{
+		onceOnLaunch = true;
 		return;
 	}
 
+	if (onceOnLaunch)
+	{
+		onceOnLaunch = false;
+		if (SConfig::GetInstance().bAutoLoadSavestate)
+			State::LoadLastSaved();
```

Comment:
I think I've tried that place. I've got stuttering sound and green screen.
After that I decided to do it on first frame.
It's the first place where the user might have loaded savestate, so it's exactly what I wanted.